.ansible-lint-ignore

```diff
@@ -0,0 +1,40 @@
+# This file contains ignores rule violations for ansible-lint
+.github/workflows/ansible-test.yml yaml[truthy]
+.github/workflows/docs-pr.yml yaml[line-length]
+.github/workflows/docs-pr.yml yaml[truthy]
+.github/workflows/docs-push.yml yaml[truthy]
+.github/workflows/import-galaxy.yml yaml[truthy]
+.github/workflows/reuse.yml yaml[truthy]
+.github/workflows/tools-tests.yml yaml[truthy]
+changelogs/config.yaml yaml[indentation]
+galaxy.yml galaxy[tags]
+galaxy.yml galaxy[version-incorrect]
```

Comment:
```suggestion
```
Please move these rules as permanent ignores to `.ansible-lint`. They make no sense and there is no reason to fix them. IMO this file should only contain things that should be fixed, not things that are disabled because they make no sense.